packages/wp-utils/src/bindToCallbacks.ts

```diff
@@ -13,11 +13,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const bindToCallbacks = (callbacks, config) => {
+
+function bindToCallbacks(callbacks: Record<string, unknown>, config: object) {
   return Object.entries(callbacks).reduce((_callbacks, [name, callback]) => {
     _callbacks[name] = callback.bind(null, config);
     return _callbacks;
   }, {});
```

Comment:
I'd also rewrite this to `Object.fromEntries` as that's a much better and cleaner way to create dynamic objects rather than reduce. Something like:

```suggestion
  return Object.fromEntries(Object.entries(callbacks).map(
    ([name, callback]) => [name, callback.bind(null, config)]
  ));
```